doc/man3/SSL_get_handshake_rtt.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+SSL_get_handshake_rtt
+- get round trip time for SSL Handshake
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_get_handshake_rtt(const SSL *s, uint64_t *rtt);
+
+=head1 DESCRIPTION
+
+SSL_get_handshake_rtt() retrieves the round-trip time (RTT) for I<ssl>.
+
+This metric is represented in microseconds (us) as a uint64_t data type.
+
+=head1 NOTES
+
+This metric is created by taking two timestamps during the handshake and providing the
+difference between these two times.
+
+When acting as the server, one timestamp is taken when the server is finished
+writing to the client. This is during the ServerFinished in TLS 1.3 and 
+ServerHelloDone in TLS 1.2. The other timestamp is taken when the server is done reading 
+the client's response. This is after the client has responded with ClientFinished.
+
+When acting as the client, one timestamp is taken when the client is finished writing 
+the ClientHello and early data (if any). The other is taken when client is done reading the server's response.
+This is after ServerFinished in TLS 1.3 and after ServerHelloDone in TLS 1.2.
+
+In addition to network propagation delay and network stack overhead, this metric 
+includes processing time on both endpoints, as this is based on TLS protocol-level 
+messages and the TLS protocol is not designed to measure network timings. In some cases the 
+processing time can be significant, especially when the processing includes asymmetric cryptographic 
+operations.
+
+=head1 RETURN VALUES
+
+Returns 1 if the TLS handshake round-trip time is successfully retrieved.
+Returns 0 if the TLS handshake round-trip time is not yet determined or if an error occurred in 
+calculation.
+
+=head1 HISTORY
+
+TLS round-trip time calculation was added in OpenSSL 3.2.0 by a Brigham Young University Capstone 
+team partnering with Sandia National Laboratories.
```

Comment:
Normally there is no info in the man pages on  who contributed what.
Better place this in `NEWS.md`.

Reply:
This is not an item for NEWS.md. I think you meant CHANGES.md. But yeah, very good point that this should have a CHANGES.md entry. Please add it.